lib/rules/no-jquery.js

```diff
@@ -1,15 +1,22 @@
 'use strict';
 
-const ember = require('../utils/ember');
-const utils = require('../utils/utils');
-const types = require('../utils/types');
+const { ReferenceTracker } = require('eslint-utils');
+const { isCallExpression } = require('../utils/types');
+const jqueryMethods = require('../utils/jquery-methods');
+
+const jqueryMap = {
+  [ReferenceTracker.CALL]: true,
+};
+
+for (const method of jqueryMethods) {
+  jqueryMap[method] = { [ReferenceTracker.READ]: true };
```

Comment:
Why not `ReferenceTracker.CALL` here like in `no-global-jquery`?

Reply:
No reason, I can update this to use `ReferenceTracker.CALL` instead